src/microbe_stage/editor/OrganellePopupMenu.cs

```diff
@@ -66,15 +71,19 @@ public bool ShowPopup
     }
 
     /// <summary>
-    ///   The placed organelle to be shown options of.
+    ///   The main organelle.
+    /// </summary>
+    public OrganelleTemplate? SelectedOrganelle { get; set; }
```

Comment:
Why is this a separate variable? Why not require the first organelle in the selected list to be the one this returns?